Projects/Claims/Sources/Views/submitClaim/Checkout/SubmitClaimCheckoutNoRepairScreen.swift

```diff
@@ -88,50 +87,14 @@ public struct SubmitClaimCheckoutNoRepairScreen: View {
                     .foregroundColor(hLabelColor.primary)
                 Spacer()
 
-                if checkIfNotDecimal(value: model?.amount ?? 0) {
-
-                    hText(
-                        prefix + formatDoubleWithoutDecimal(value: model?.amount ?? 0) + " "
-                            + String(model?.currencyCode ?? "")
-                    )
-                    .foregroundColor(hLabelColor.secondary)
-                } else {
-                    hText(
-                        prefix + formatDoubleWithDecimal(value: model?.amount ?? 0) + " "
-                            + String(model?.currencyCode ?? "")
-                    )
-                    .foregroundColor(hLabelColor.secondary)
-                }
+                hText(
+                    prefix + (model?.getAmountWithCurrency() ?? "")
```

Comment:
Instead of using prefix here, couldn't we just negate the amount? And pass a negative amount to the formatter then it should add the `-1000kr` automatically 😄

Reply:
@sampettersson The thing is that we then have to "modify" the values that we get from the backend to be negative which can cause problems in the future if we need to use these values somewhere else  😅

Reply:
I wouldn't store them as negative, that I agree with can cause problems, but if we know that something always means negative from a user POV, like a deductible or a discount, then I think it makes sense to do that conversion in the view layer, totally agree to not store it like a negative amount in the Store.